### PR TITLE
Change Poltergeist's activation method

### PR DESCRIPTION
**Pull request by KrisXV**
Server-side part of https://github.com/smogon/pokemon-showdown-client/pull/1598

### CODE REVIEW

**Comment by Zarel** [comment]
No, this should be an `-activate`. We used to be inconsistent about this, but in the new protocol, `-item` is only for item changes, `-activate` is for item reveals.

File: data/moves.ts [hunk 1]
```diff
@@ -13714,7 +13714,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 			}
 		},
 		onTryHit(target, source, move) {
-			this.add('-activate', target, 'move: Poltergeist', this.dex.getItem(target.item).name);
+			this.add('-item', target, this.dex.getItem(target.item), '[from] move: Poltergeist', '[of] ' + source);
```
Comment (Zarel) [inline data/moves.ts:13717]:
```suggestion
			this.add('-activate', target, this.dex.getItem(target.item).fullName, '[from] move: Poltergeist', '[of] ' + source);
```

Something like _this_ I think might be closer to what you want, though.

Comment (Zarel) [reply data/moves.ts:13717]: Although the old version should in theory also work. Is there a reason you wanted to change it?

Comment (KrisXV) [reply data/moves.ts:13717]: no the server-side is useless, my bad

**Pull request closed without merging**